appcues/src/main/java/com/appcues/data/model/ExperienceStepFormState.kt

```diff
@@ -82,6 +85,17 @@ internal sealed class ExperienceStepFormItemState(
     val attributeName: String?,
 ) {
 
+    companion object {
+
+        private const val ITEM_TYPE_TEXT_INPUT = "textInput"
+        private const val ITEM_TYPE_OPTION_SELECT = "optionSelect"
+    }
+
+    val isTextFocusable: Boolean
+        get() {
+            return type == ITEM_TYPE_TEXT_INPUT
+        }
```

Comment:
Could just make this prop part of the `ExperienceStepFormItemState` constructor and require implementations to pass it in? so `TextInputFormItemState` would pass true.

Just thinking of the future if/when we add another type, and might forget to come back here and update this other line of code.